### PR TITLE
[13.0][MIG] business_requirement: Migration to 13.0

### PR DESCRIPTION
**Pull request by Tardo**
cc @tecnativa TT19716

### CODE REVIEW

File: business_requirement/views/business_view.xml [hunk 1]
```diff
@@ -251,7 +251,7 @@
         <field name="name">Business Requirement</field>
         <field name="res_model">business.requirement</field>
         <field name="view_mode">tree,kanban,form,graph,pivot</field>
-        <field name="view_type">form</field>
+        <field name="binding_view_types">form</field>
```
Comment (pedrobaeza) [inline business_requirement/views/business_view.xml]: You must remove this, as this is for binding an action in the dropdown menu "Actions".

File: business_requirement/views/business_view.xml [hunk 2]
```diff
@@ -260,7 +260,7 @@
         <field name="name">Categories</field>
         <field name="res_model">business.requirement.category</field>
         <field name="view_mode">tree,form,graph</field>
-        <field name="view_type">form</field>
+        <field name="binding_view_types">form</field>
```
Comment (pedrobaeza) [inline business_requirement/views/business_view.xml]: You must remove this, as this is for binding an action in the dropdown menu "Actions".

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by Tardo** [comment]
Ready to review

**Review by pedrobaeza — APPROVED** [review]

**Comment by Tardo** [comment]
Need reviews!

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**